x-pack/plugins/spaces/server/lib/get_active_space.js

```diff
@@ -8,13 +8,13 @@ import Boom from 'boom';
 import { wrapError } from './errors';
 import { getSpaceIdFromPath } from './spaces_url_parser';
 
-export async function getActiveSpace(savedObjectsClient, requestBasePath, serverBasePath) {
+export async function getActiveSpace(spacesClient, requestBasePath, serverBasePath) {
```

Comment:
I'm not sure if the "not found" handling below will work as intended, because the Spaces Client throws an error if the requested space can't be found:

```javascript
if (!space) {
     throw Boom.notFound(
       `The Space you requested could not be found. Please select a different Space to continue.`
    );
  }
```

I think we need to move this into the catch block and check if the error is the "not found" error....or remove it altogether